tests/integration/replication.tcl

```diff
@@ -70,11 +70,25 @@ start_server {tags {"repl"}} {
             }
         }
 
-        test {INCRBYFLOAT replication, should not remove expire} {
```

Comment:
We don't want this test anymore?

Reply:
OMG, i guess i deleted it while debugging my code and didn't notice it when submitting the PR.